portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase

```diff
@@ -73,6 +73,75 @@ definition {
 		BlogsEntry.viewInlineImage(uploadFileName = "Document_1");
 	}
 
+	@priority = "4"
+	test AddMBThreadWithLinkToDocument {
+		property portal.acceptance = "true";
+
+		JSONDocument.addFileWithUploadedFile(
+			dmDocumentDescription = "DM Document Description",
+			dmDocumentTitle = "Document_1.jpg",
+			groupName = "Guest",
+			mimeType = "image/jpeg",
+			sourceFileName = "Document_1.jpg");
+
+		MessageBoardsAdmin.openMessageBoardsAdmin(siteURLKey = "guest");
+
+		LexiconEntry.gotoAdd();
```

Comment:
What's a `LexiconEntry`?

Reply:
good question, I'm not exactly sure myself since to me it implies something related to Lexicon...but looking over the macros in LexiconEntry, it looks like they're actions that can be done in a generic admin control panel page like Control Panel > Blogs. I think the intent was to have something generic that refers to common elements that all/most component teams are using (it's now evolved to clay components afaik?). In my opinion, I don't like seeing `Lexicon` being used in poshi naming and probably needs to be updated...I pointed this out during our FI path refactor and I think references to Lexicon should be removed.